packages/angular/cli/lib/config/schema.json

```diff
@@ -398,6 +401,33 @@
         "^[a-z]{1,3}-.*": {}
       },
       "definitions": {
+        "i18n": {
+          "description": "Project i18n options",
+          "type": "object",
+          "properties": {
+            "missingTranslation": {
+              "type": "string",
+              "description": "How to handle missing translations for i18n.",
+              "enum": ["warning", "error"]
+            },
+            "sourceLocale": {
+              "type": "string",
+              "description": "The locale id used in the source code.",
+              "default": "en-US"
+            },
+            "locales": {
```

Comment:
@clydin should we also add an `inlineLocales` option at this level?

Reply:
That should be an option in the build itself.  The goal here is to define the locales.  How they are used (inlined, processed/copied to the output for runtime i18n, etc.) is builder behavior.